climakitae/unit_conversions.py

```diff
@@ -0,0 +1,74 @@
+"""
+This script calculates alternative units for variables with multiple commonly used units.
+
+"""
+
+def _convert_units(da, selected_units):
+    """ Converts units for any variable
+
+    Args:
+      da (xr.DataArray): Data
+      selected_units (str): Selected units of data, from selections.units
+
+    Returns:
+      da (xr.DataArray): Data with converted units and updated units attribute
+    """
+    
+    # Get native units of data from attributes 
+    try: 
+        native_units = da.attrs["units"] 
+    except: 
+        raise ValueError("You've encountered a bug in the code. This variable does not have identifiable native units. The data for this variable will need to have a 'units' attribute added in the catalog.")
+
+    # Rename poorly formatted native units for better readability
+    if native_units == "kg kg-1": 
+        da.attrs["units"] = "kg/kg"
+        native_units = "kg/kg" 
+    if native_units == "m s-1": 
+        da.attrs["units"] = "m/s"
+        native_units = "m/s" 
+    
+    # Pass if chosen units is the same as native units
+    if native_units == selected_units:
+        pass
+
+    # Precipitation units
+    elif native_units == "mm":
+        if selected_units == "inches":
+            da = da / 25.4
+            da.attrs["units"] = selected_units
+
+    # Moisture ratio units
+    elif native_units == "kg/kg":
+        if selected_units == "g/kg":
+            da = da * 1000
+            da.attrs["units"] = selected_units
+
+    # Pressure units
+    elif native_units == "Pa":
+        if selected_units == "hPa":
+            da = da / 100.
+            da.attrs["units"] = selected_units
+        elif selected_units == "mb":
+            da = da / 100.
+            da.attrs["units"] = selected_units
+        elif selected_units == "inHg":
+            da = da / 3386.39
```

Comment:
was there any discussion about precision for these? I think this one is an approximation, and just want to check that we are intentionally choosing our number of sig figs!

Reply:
This is a very good catch @ckingdon95! This conversion matches what the NWS uses in their own calculations (https://www.weather.gov/media/epz/wxcalc/pressureConversion.pdf), but is the opposite direction (will fix now!) so I would feel confident in also doing so, once I've fixed it.

Reply:
Ah okay great! (I think you have the direction correct here though?) And maybe we could add links to these sources as a comment at the top of the file?

Reply:
@ckingdon95 and I confirmed that multiplying by 0.000295300 is the same operation as dividing by 3386.39, but for our purposes to better follow NWS guidelines that the multiplication option is the way to go. 

Links have been put in the docstring for documentation purposes.